Lib/subprocess.py

```diff
@@ -260,8 +260,20 @@ def _args_from_interpreter_flags():
         v = getattr(sys.flags, flag)
         if v > 0:
             args.append('-' + opt * v)
-    for opt in sys.warnoptions:
+
+    if hasattr(sys, '_xoptions'):
```

Comment:
Do we want to add other `-X` options, such as `-Xfaulthandler`?

Reply:
Sure! Done.